Formula/l/logtalk.rb

```diff
@@ -24,5 +24,8 @@ class Logtalk < Formula
 
   def install
     cd("scripts") { system "./install.sh", "-p", prefix }
+    ohai "To complete the Logtalk installation, define the environment variables:"
+    ohai "    LOGTALKHOME=#{HOMEBREW_PREFIX}/share/logtalk"
+    ohai "    LOGTALKUSER=$HOME/logtalk"
```

Comment:
this should be moved to caveats

Reply:
Done. Thanks for the feedback.

May I take this opportunity to ask you a question about `.info` files? The formula calls an installation script that creates a `share/info` directory containing two `.info` files. But after `brew install`, those files are not linked from `/usr/local/share/info`. Am I missing some step?